packages/commonwealth/server/controllers/server_communities_methods/update_community.ts

```diff
@@ -215,6 +227,32 @@ export async function __updateCommunity(
   if (directory_page_chain_node_id !== undefined) {
     chain.directory_page_chain_node_id = directory_page_chain_node_id;
   }
+  if (namespace !== undefined) {
+    if (!transactionHash) {
+      throw new AppError(Errors.InvalidTransactionHash);
+    }
+
+    if (!ownerOfChain) {
+      throw new AppError(Errors.NotAdmin);
+    }
+
+    // TODO: Dirty hack. Namespace chain_node_id error checking should be runtime, not static time error check.
+    //  Refactor with validateNamespace and generalize when we have more time or support more chains
+    if (chain.chain_node_id !== 1263) {
```

Comment:
Why do we need to filter here at all? Can't we let validateNamespace do the work?